Fix the following error:

main.c: In function 'main_':
main.c:595:22: error: variable 'subpkg_tried' might be clobbered by 'longjmp' or 'vfork' [-Werror=clobbered]
  595 |                 bool subpkg_tried = false;
      |                      ^~~~~~~~~~~~
--- micropython-1.14.orig/ports/unix/main.c
+++ micropython-1.14/ports/unix/main.c
@@ -592,7 +592,7 @@
 
                 mp_obj_t mod;
                 nlr_buf_t nlr;
-                bool subpkg_tried = false;
+                static bool subpkg_tried = false;
 
             reimport:
                 if (nlr_push(&nlr) == 0) {
